agent/src/agent/pipeline/config/stages/source/influx.py

```diff
@@ -26,7 +26,10 @@ def get_auth_config(self):
             'conf.client.basicAuth.password': self.pipeline.source.config.get('password', '')
         }
 
-    def get_query(self):
+    def get_query(self) -> str:
+        if self.pipeline.query:
+            return f"{quote_plus(self.pipeline.query)}"
```

Comment:
we should add the timestamp condition so pipeline will be able to fetch them continuously. Like in jdbc pipelines, require a client to have a {TIMESTAMP_CONDITION} constant in the query and replace it with `("time" > ${record:value('/last_timestamp')} AND "time" <= ${record:value('/last_timestamp')} + {interval} AND "time" < now() - {delay})"

Reply:
Updated.
Maybe we need to set manually these **meas1, meas2, meas3** (from your example) values as  `measurement_name` in pipeline config so they appear in next JS stages

Because now, for example, records in JS tranform stage look as following (measurement_category - empty):

```text
properties : MAP
   target_type : gauge
   test : val
   cpu : cpu-total-usage
   host : msk-air-trn5
   measurement_category : {}
   what : usage_active
```